source/extensions/load_balancing_policies/client_side_weighted_round_robin/client_side_weighted_round_robin_lb.h

```diff
@@ -0,0 +1,211 @@
+#pragma once
+
+#include "envoy/extensions/load_balancing_policies/client_side_weighted_round_robin/v3/client_side_weighted_round_robin.pb.h"
+#include "envoy/upstream/upstream.h"
+
+#include "source/extensions/load_balancing_policies/common/load_balancer_impl.h"
+
+#include "absl/status/status.h"
+
+namespace Envoy {
+namespace Upstream {
+
+using ClientSideWeightedRoundRobinLbProto = envoy::extensions::load_balancing_policies::
+    client_side_weighted_round_robin::v3::ClientSideWeightedRoundRobin;
+using OrcaLoadReportProto = xds::data::orca::v3::OrcaLoadReport;
+
+/**
+ * Load balancer config used to wrap the config proto.
+ */
+class ClientSideWeightedRoundRobinLbConfig : public Upstream::LoadBalancerConfig {
+public:
+  ClientSideWeightedRoundRobinLbConfig(const ClientSideWeightedRoundRobinLbProto& lb_config,
+                                       Event::Dispatcher& main_thread_dispatcher)
+      : lb_config_(lb_config), main_thread_dispatcher_(main_thread_dispatcher) {}
+
+  const ClientSideWeightedRoundRobinLbProto lb_config_;
```

Comment:
nit: I personally think to load the configuration from the proto to specific configuration like `blackout_period_` here would be better. Then, when you creating the LB, you needn't to access the proto.